pkgs/applications/version-management/gittyup/default.nix

```diff
@@ -0,0 +1,117 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmark
+, mount
+, http-parser
+, perl
+, cmake
+, qtbase
+, qttools
+, git
+, git-lfs
+, ninja
+, libssh2
+, libgit2
+, pkg-config
+, libsecret
+, libgnome-keyring
+, pcre
+, wrapQtAppsHook
+, makeDesktopItem
+}:
+stdenv.mkDerivation rec {
+  pname = "gittyup";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "Murmele";
+    repo = "Gittyup";
+    rev = "gittyup_v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-WJzw1uFCE5dyI1F0FEzww05zCsYPEgeXBz+HogTYX9w=";
+  };
+
+  nativeBuildInputs = [
+    cmark
+    mount
+    http-parser
+    perl
+    cmake
+    qttools
+    ninja
+    pkg-config
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtbase
+    git
+    git-lfs
+    libssh2
+    libgit2
+    libsecret
+    libgnome-keyring
+    pcre
```

Comment:
You are sure about those? On first sight I didn't see dependencies for this in the repo.